easybuild/easyblocks/generic/perlmodule.py

```diff
@@ -53,6 +53,10 @@ def __init__(self, *args, **kwargs):
         """Initialize custom class variables."""
         super(PerlModule, self).__init__(*args, **kwargs)
         self.testcmd = None
+        # Environment variables PERL_MM_OPT and PERL_MB_OPT cause installations to fail. 
```

Comment:
@RvDijk Let's move the extra line you added below this block to above this block? :)

Also, drop the `WIP` from the PR title, it looks good to go.